contracts/VotingEscrow.vy

```diff
@@ -87,6 +89,7 @@ MULTIPLIER: constant(uint256) = 10 ** 18
 
 token: public(address)
 supply: public(uint256)
+is_active: public(bool)
```

Comment:
```
/* Fees breaker, to protect withdraws if anything ever goes wrong */
breaker: public(bool)
```

Something like we used in the early liquidity mining days
https://etherscan.io/address/0xCcD5cb3401704AF8462a4FFE708a180d3C5c4Da0#code